packages/primitives/src/svg/index.native.js

```diff
@@ -39,7 +49,7 @@ export const SVG = ( { className = '', isPressed, ...props } ) => {
 	const appliedProps = { ...props, style: styleValues };
 
 	return (
-		<Svg
+		<AnimatedSvg
```

Comment:
I wonder if we should have all SVGs as animated components or just when we actually want to animate something. Do you think this would affect the performance in some way? By having all icons as Animated components?

Reply:
It's shouldn't affect, however for the sake of calm I will create a separate component for it. Thanks!

Reply:
Awesome! Thanks!